test/server/util/twilio_mocks.js

```diff
@@ -22,25 +25,70 @@ export function callStatusUpdate() {
     priceUnit: 'USD',
     sid: 'CAcbbf06f666c72c51c59200de56ae54ff',
     startTime: '2017-09-11T18:20:30.000Z',
-    status: 'completed',
+    status: null,
     subresourceUris: {
       notifications: `/2010-04-01/Accounts/${TWILIO_ACCOUNT_SID}/Calls/CAcbbf06f666c72c51c59200de56ae54ff/Notifications.json`,
       recordings: `/2010-04-01/Accounts/${TWILIO_ACCOUNT_SID}/Calls/CAcbbf06f666c72c51c59200de56ae54ff/Recordings.json`
     },
     to: '+14086428264',
     toFormatted: '(408) 642-8264',
     uri: `/2010-04-01/Accounts/${TWILIO_ACCOUNT_SID}/Calls/CAcbbf06f666c72c51c59200de56ae54ff.json`
-  };
+  }
```

Comment:
It's typically regarded as a 'best-practices' to avoid switch statements, though in this case I think it's fine. I would add 'break' statements after cases, though, which will prevent fall-through. 
(side-note: I know redux doesn't adhere to any of this, and in reducers breaks are *not* used, but I think that's b/c of how the store construction/root reducer works).

Reply:
Oh, this comment is for the code below, on line 37.